cumulusci/utils/deprecation.py

```diff
@@ -0,0 +1,13 @@
+import os
+from warnings import warn
+
+
+def warn_moved(new, old):
+    msg = f"Deprecation warning: Please import from {new} rather than {old}"
+
+    # this is a hard-error if encountered in testing context
+    if "PYTEST_CURRENT_TEST" in os.environ:
+        assert 0, msg
```

Comment:
It would break downstream code if that code needed to be fixed. I'm not convinced that's a bad thing given the totality of our context (how many people use CCI as a library, how many of them are not on our team, how many are likely using the old paths, ...)